services/movie_session.py

```diff
@@ -42,3 +42,11 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
```

Comment:
Consider handling the `MovieSession.DoesNotExist` exception in `delete_movie_session_by_id` to manage cases where the session with the specified ID does not exist. This will prevent the application from crashing due to unhandled exceptions.